cli/cmd/version.go

```diff
@@ -13,30 +13,65 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	cliFlag     = "cli"
+	clusterFlag = "cluster"
+)
+
 var (
-	OdigosVersion string
-	OdigosCommit  string
-	OdigosDate    string
+	OdigosVersion        string
+	OdigosClusterVersion string
+	OdigosCommit         string
+	OdigosDate           string
 )
 
 // versionCmd represents the version command
 var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print odigos version.",
 	Run: func(cmd *cobra.Command, args []string) {
-		short_flag, _ := cmd.Flags().GetBool("short")
+		cliFlag, _ := cmd.Flags().GetBool(cliFlag)
+		clusterFlag, _ := cmd.Flags().GetBool(clusterFlag)
 
-		if short_flag {
+		if cliFlag {
 			fmt.Printf("%s\n", OdigosVersion)
+		}
```

Comment:
@blumamir I changed it so you can do multiple arguments (cli + cluster).. I'm not sure there is or will be a use case for it. But, usually CLIs allow multiple arguments. So even in the future if we will want to add